docs/docs/operate/profiling.md

```diff
@@ -0,0 +1,48 @@
+Mantis allows you to directly connect profiling tools to workers in order to do performance or memory profiling. This
+can be helpful for debugging OOMs. Below are the steps to start profiling your running Mantis job:
+
+ 1. Download a profiling tool. We recommend [Zulu Mission Control (ZMC)](https://www.azul.com/products/zulu-mission-control/).
+ 1. Untar ZMC
+    ```
+    tar -xzvf <zulu_mission_control_package>.tar.gz
+    ```
+ 1. Run zmc.
+    ```
+    note
```

Comment:
Does this work? I think it needs to be `!!! note`

Reply:
Thanks for catching this, fixing